config/jobs/image-pushing/k8s-staging-cluster-api.yaml

```diff
@@ -57,7 +57,7 @@ postsubmits:
       decorate: true
       branches:
         - ^master$
-        - ^release-0.3$
+        - ^release-0.4$
```

Comment:
I think we still want to include release-0.4 actually, since we'll be cutting patch releases from there

Reply:
so keep `release-0.4`? or do you mean keep `release-0.3`

Reply:
added `release-0.3` as well as `release-0.4`

Reply:
we might not need this change actually, since the tag should trigger an image build with the semver regex below. Looking at CAPI and CAPA they don't have latest release branches...